modules/xmpp/JingleSessionPC.js

```diff
@@ -56,9 +57,8 @@ function JingleSessionPC(me, sid, peerjid, connection,
     // stable and the ice connection state is connected.
     this.modifySourcesQueue.pause();
 }
-//XXX this is badly broken...
-JingleSessionPC.prototype = JingleSession.prototype;
-JingleSessionPC.prototype.constructor = JingleSessionPC;
+
+util.inherits(JingleSessionPC, JingleSession);
```

Comment:
Is that the first use of util.inherits in lib-jitsi-meet? If that's the case why do it for JingleSession(PC) only and not do it everywhere we extend classes in order to not introduce inconsistencies?